compiler/errors.cpp

```diff
@@ -37,14 +37,14 @@
   #include <alloc/fortify.h>
 #endif
 #include "sc.h"
-#include "sc5.h"
+#include "errors.h"
 
 #if defined _MSC_VER
   #pragma warning(push)
   #pragma warning(disable:4125)  /* decimal digit terminates octal escape sequence */
 #endif
 
-#include "sc5-in.scp"
+#include "messages.scp"

```

Comment:
while we're doing renaming, is there any reason .scp is preferred over .cpp?

Reply:
Probably a better name would be ".tbl" or something. It's not really a typical source file or header. I've no idea what "scp" meant.

Reply:
Oh, these things actually *are* headers... guess they should be .h then.